### ISSUE TITLE
on new book,pages etc, set title input to autofocus

### ISSUE THREAD

**Issue by ChangePlaces** [BookStackApp/BookStack#1956]
it's annoying having to use the mouse to click in the title input to then type! This should be a general feature across all 'new' forms

**Comment by ssddanbrown** [comment]
@ChangePlaces Thanks for the suggestion. Accessibility concerns will need to be considered with such a change.

**Comment by ssddanbrown** [comment]
Thanks again @ChangePlaces. Pages already had this feature at time of writing but this has now been added to shelves, books and chapters. Will be part of the next feature release.
